### PR TITLE
Changes to Lum

### PR DESCRIPTION
**Pull request by jmcclain1299**
Lum should work on Haban now

### CODE REVIEW

**Comment by jmcclain1299** [comment]
updated to add the changes recently made to the base game I think?

File: src/scripts/farming/aura/Aura.ts [hunk 1]
```diff
@@ -13,7 +13,9 @@ class Aura {
         if (plot.stage() < PlotStage.Taller) {
             return;
         }
-        const multiplier = this.auraMultipliers[plot.stage() - 2] * plot._auras[AuraType.Boost]();
+        const berryAura = this.auraMultipliers[plot.stage() - 2]; /*this should make typing easier*/
```
Comment (RedSparr0w) [inline src/scripts/farming/aura/Aura.ts:16]:
```suggestion
        const berryAura = this.auraMultipliers[plot.stage() - 2];
```

File: src/scripts/farming/Plot.ts [hunk 2]
```diff
@@ -194,9 +194,12 @@ class Plot implements Saveable {
             }
 
             // Aura
+
             if (this.stage() >= PlotStage.Taller && this.berryData.aura) {
+                const berryAuraValue = this.berryData.aura.getAuraValue(this.stage());
+                const lumAuraValue = this._auras[AuraType.Boost]();
                 tooltip.push('<u>Aura Emitted:</u>');
-                const emittedAura = this.berryData.aura.getAuraValue(this.stage()) * this._auras[AuraType.Boost]();
+                const emittedAura = (berryAuraValue >= 1) ? (berryAuraValue * lumAuraValue) : (berryAuraValue / lumAuraValue); /* I fought this forever wondering why the tooltip wasn't right */
```
Comment (RedSparr0w) [inline src/scripts/farming/Plot.ts:202]:
```suggestion
                const emittedAura = (berryAuraValue >= 1) ? (berryAuraValue * lumAuraValue) : (berryAuraValue / lumAuraValue);
```

**Review by RedSparr0w — APPROVED** [review]
Looks good, thanks for the PR! 👍

**Pull request merged**